org.gnome.Keysign.yml

```diff
@@ -0,0 +1,306 @@
+id: org.gnome.Keysign
+branch: master
```

Comment:
It should be `stable` or removed completely.

Reply:
Removed with my last commit.